.github/workflows/image_base.yml

```diff
@@ -18,17 +18,17 @@ jobs:
         uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-      - name: Login to Quay
```

Comment:
open this PR for code review, if the code is good enough, will change back on CI settings as secret usage and remove hook in PR.